src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -1,17 +1,28 @@
 <template>
   <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="add-notation-dialog">
     <v-card>
-      <v-card-title id="dialog-title">Add a {{displayName}}</v-card-title>
+      <v-card-title id="dialog-title">
+        <span v-if="administrativeDissolution" id="dialog-title"><strong>{{displayName}}</strong></span>
+        <span v-else-if="putBackOn" id="dialog-title"><strong>Correction - {{displayName}}</strong></span>
+        <span v-else id="dialog-title"><strong>Add a {{displayName}}</strong> </span>
+      </v-card-title>
       <v-card-text>
+        <div id="dialog-text" class="dialog-text">
+          <p v-if="administrativeDissolution"> You are about to dissolve
+          <strong><span class="text-uppercase">{{getEntityName}}</span>, {{getIdentifier}}</strong> . </p>
+          <p v-if="putBackOn"> You are about to put <strong><span class="text-uppercase">{{getEntityName}}</span>,
+          {{getIdentifier}}</strong> back on the register.</p>
+        </div>
         <div id="notation-text" class="mb-4 mt-2">
-          Enter a {{displayName}} that will appear on the ledger for this entity
+          Enter a {{(administrativeDissolution || putBackOn) ? 'Detail' : displayName}}
+          that will appear on the ledger for this entity
         </div>
         <v-form ref="notationFormRef" v-model="notationFormValid" id="notation-form">
           <v-textarea
             v-model="notation"
             class="text-input-field mb-2"
             filled
-            :label="displayName"
+            :label="administrativeDissolution || putBackOn? 'Add Detail' : displayName"
```

Comment:
You have this same logic on line 17. Would it be possible to remove from both places and add a local getter instead?

Reply:
I was thinking it would be a getter for the string itself. `notationLabel` maybe?

(Lines 17 and 25 aren't consistent in what is displayed in the else case, so I leave it to you to work that out.)

In case you're stuck and there is higher-priority work to do, I think this isn't serious to block this PR. If you decide not to do this now then just add a comment to that effect here.

Reply:
Looks good.